nautobot/ipam/tests/test_api.py

```diff
@@ -348,6 +348,8 @@ def setUpTestData(cls):
         cls.create_data = [
             {
                 "prefix": "192.168.4.0/24",
+                "network": "192.168.4.0",  # not required, but possible to specify
+                "broadcast": "192.168.4.255",  # not required, but possible to specify
```

Comment:
Adding these here was easier than updating the enhanced `test_update_object` test case to understand that setting the `prefix` field in a PATCH would automatically change the `network` and `broadcast` fields correspondingly.